files/templates/additional-scrape-configs.template.yaml

```diff
@@ -828,10 +828,18 @@
   relabel_configs:
   - source_labels: [__meta_kubernetes_service_annotationpresent_giantswarm_io_monitoring]
     regex: (true)
-    action: keep
-  - source_labels: [__meta_kubernetes_service_annotation_giantswarm_io_monitoring]
+    target_label: __tmp_keep_me
```

Comment:
Why do we need this `__tmp_keep_me` label ?

Reply:
This allows us to scape both managed apps with the giantswarm.io/monitoring=true annotations and labels. This is basically a logical OR in relabelling syntax